packages/config/maintenance/importConfig.ts

```diff
@@ -865,11 +884,16 @@ async function parseZWAFiles(): Promise<void> {
 		const manufacturerName =
 			configManager.lookupManufacturer(manufacturerId);
 
-		// Add the manufacturer to our manufacturers.json if it is missing
-		if (Number.isNaN(manufacturerId)) {
-		} else if (manufacturerName === undefined && file.Brand !== undefined) {
-			console.log(`Adding missing manufacturer: ${file.Brand}`);
-			configManager.setManufacturer(manufacturerId, file.Brand);
+		if (file.ManufacturerId != "0x9999") {
+			// Add the manufacturer to our manufacturers.json if it is missing
+			if (Number.isNaN(manufacturerId)) {
+			} else if (
```

Comment:
```suggestion
			if (
				!Number.isNaN(manufacturerId) &&
```
feels a bit more natural than an empty if

Reply:
This was already there. I assumed you did that for a reason. I thought it was odd too. I'll change it.

Reply:
Dunno, maybe the condition evolved over time